src/Analysis/Ast/Test/FluentAssertions/MemberAssertions.cs

```diff
@@ -61,20 +61,20 @@ internal class MemberAssertions : ReferenceTypeAssertions<IMember, MemberAsserti
             return new AndWhichConstraint<MemberAssertions, IMember>(this, Subject);
         }
 
-        public AndWhichConstraint<MemberAssertions, IPythonClassType> HaveBase(string name, string because = "", params object[] reasonArgs) {
+        public AndWhichConstraint<MemberAssertions, IPythonType> HaveBase(string name, string because = "", params object[] reasonArgs) {
```

Comment:
If you want, you can borrow https://github.com/MikhailArkhipov/python-language-server/blob/functools/src/Analysis/Ast/Impl/Specializations/Typing/Types/GenericClassBase.cs which is former `GenericTypeParameter` and is actually a class - as it is supposed to be.

Reply:
If we just want to check if a base is inside the Bases list, then we should just need to pass in the name of the base. By having it as PythonClassType, we can have cases with typing or builtins where the type is a PythonType but we look for PythonClassType. Unless we change Bases to a list of IPythonClassTypes, then I think this is clearer.